.github/renovate.json5

```diff
@@ -0,0 +1,5 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": ["github>canonical/data-platform//renovate_presets/charm.json5"],
```

Comment:
We may want to have a separate preset for rocks in the future.

Reply:
I don't know much about the renovate syntax, but I was wondering if the double slash is right at `"github>canonical/data-platform//renovate_presets/charm.json5"`.

Reply:
IIRC it indicates the directory path in the repo.